qiskit/synthesis/unitary/aqc/__init__.py

```diff
@@ -108,7 +108,7 @@
     unitary = ...
 
     # Define a number of qubits for the algorithm, at least 3 qubits
-    num_qubits = int(round(np.log2(unitary.shape[0])))
+    num_qubits = int(round(math.log2(unitary.shape[0])))
```

Comment:
`int(round(...))` seems very wrong, but that's extra this PR

Reply:
"extra _to_ this PR", I mean - no need to change it.

Reply:
dropped the int since its superfluous in af57ee4cb4c421d922c16b22cf69a1bc85efdf98

Reply:
Fixed it anyway  in 9588d6c9e